code/controllers/subsystem/minimap.dm

```diff
@@ -338,6 +338,170 @@ SUBSYSTEM_DEF(minimaps)
 	hashed_minimaps[hash] = map
 	return map
 
+/**
+ * Fetches the datum containing an announced flattend map png reference.
+ *
+ * Arguments:
+ * * faction: FACTION_MARINE or XENO_HIVE_NORMAL
+ */
+/proc/get_tacmap_data_png(faction)
+	var/list/map_list
+
+	if(faction == FACTION_MARINE)
+		map_list = GLOB.uscm_flat_tacmap_data
```

Comment:
I do not recommend it until other factions are more common in the main loop, or there's a different game mode that is used enough that could benefit from command/queen outlining more of a plan.

- Its intentional that the data is not sent to the other factions to avoid finagling some way to get the enemy tactical map (but who knows whats possible when you're going to those lengths or if what I did to try to separate the data from marines <-> xenos by not entering the opposite faction in the client list for ssassets even works how I intended).

- Additionally, ghosts really ought to then have a single unified view tacmap interface like admins do. I'd prefer not to give ghosts access to the admin panel even if restricted to keep better separation; but right now they just have two verbs that open that faction's interface as if they were a player (just with more limited functionality - a single page).